ivy/functional/ivy/nest.py

```diff
@@ -1015,6 +1015,49 @@ def nested_map(
         x following the applicable of fn to it's nested leaves, or x itself if x is not
         nested.
 
+    Examples
+    --------
+    With :code:`ivy.Array` inputs:
+
+    >>> def special_square(nested : float) -> float : return np.square(x)
```

Comment:
You've defined your function as using `nested` as argument, so it should return `np.square(nested)`. Otherwise set the input as `x : float`

Reply:
changed it to x everywhere